devices/org.osbuild.loopback

```diff
@@ -35,7 +35,7 @@ SCHEMA = """
 "properties": {
   "filename": {
     "type": "string",
-    "description": "File to associate with the loopback device"
+    "description": "File to associate with the loopback device, optionally starts with input:// to use a different pipeline's tree"
```

Comment:
I'd sugges to frame this as "The file can be passed as a stage input and referenced accordingly`.